algosdk/v2client/algod.py

```diff
@@ -117,17 +117,22 @@ def pending_transactions_by_address(self, address, limit=0, response_format="jso
             **kwargs)
         return res
 
-    def block_info(self, block, response_format="json", **kwargs):
+    def block_info(self, block=None, response_format="json", round=None, **kwargs):
         """
         Get the block for the given round.
 
         Args:
             block (int): block number
-            response_format (str): the format in which the response is returned: either
-                "json" or "msgpack"
+            response_format (str): the format in which the response is 
+                returned: either "json" or "msgpack"
+            round (int, optional): alias for block; only specify one of these
         """
         query = {"format": response_format}
-        req = "/blocks/" + str(block)
+        req = "/blocks/"
+        if block is not None:
```

Comment:
Maybe this should raise an exception if both `block` and `round` are provided?
Maybe this should be a function as it's used several time?